Assets/SEE/Controls/SettingsMenue.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
+namespace SEE.Controls
+{
+    /// <summary>
+    /// Handles the actions in the
+    /// settings canvas.
+    /// </summary>
+    public class SettingsMenue : MonoBehaviour
+    {
+        //The text box, which will display the text
```

Comment:
Please use [proper C# documentation comments](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/) to document properties and methods.